candidates/models/db.py

```diff
@@ -51,7 +51,7 @@ class LoggedAction(models.Model):
     person = models.ForeignKey(Person, blank=True, null=True)
     action_type = models.CharField(max_length=64)
     popit_person_new_version = models.CharField(max_length=32)
-    created = models.DateTimeField(auto_now_add=True)
+    created = models.DateTimeField(auto_now_add=True, db_index=True)
```

Comment:
It's a bit confusing having the model change in this commit and the migration in the next. I think I'd create a separate commit that just has this model change + migration.